test/NuGet.Services.Staging.EndToEndTests/StagingClient/IStagingClient.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace NuGet.Services.Staging.EndToEndTests

```

Comment:
Should this (and implementation) go to `NuGet.Services.Staging.Client`? (maybe as a separate PR)